src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -126,6 +149,8 @@ const pageTypeUrls = async (
           apiContext: 'secondary_data',
         },
       ];
+    case ARTICLE_PAGE:
+      return getRecommendations(service, assetUri);
```

Comment:
Just thinking, we'll need to wrap this in a similar check to the STORY_PAGE case, e.g:
```
        ...((await hasRecommendations(service, variant, pageData))
          ? getRecommendations(service, assetUri)
          : []),
```

The `hasRecommendations` function checks the service config to ensure the service should attempt to fetch for recommendations. Without this, we'll be fetching for recommendations on services that don't have them yet. The function also checks for the `allowAdvertising` flag. This is available in Optimo pageData, but at a slightly different path to CPS:

CPS: `['metadata', 'options', 'allowAdvertising']`
Optimo: `['metadata', 'allowAdvertising']`

So we'll need to update that to account for both page types too.